examples/README.md

```diff
@@ -0,0 +1,61 @@
+﻿# Common switches
+
+OpenThread allows you to [configure](https://openthread.io/guides/build?hl=en#configuration) the stack for different functionality and behavior. This configuration is based on changing compile-time constants during the build process using common switches listed in `/examples/common-switches.mk`.
+
+This page lists the available common switches with description and additional information. For build command examples, see [build examples](https://openthread.io/guides/build?hl=en#build_examples).
+
+| Makefile switch | CMake switch | Default value | Description | Additional information |
+| --- | --- | --- | --- | --- |
+| BACKBONE_ROUTER | OT_BACKBONE_ROUTER | 0 | Enables Backbone Router functionality for Thread 1.2. |  |
+| BIG_ENDIAN | OT_BIG_ENDIAN | 0 | Allows the host platform to use big-endian byte order. |  |
+| BORDER_AGENT | OT_BORDER_AGENT | 0 | Enables support for border agent. | In most cases, enable this switch if you are building On-mesh Commissioner or Border Router with External Commissioning support. |
+| BORDER_ROUTER | OT_BORDER_ROUTER | 0 | Enables support for Border Router. | This switch is usually combined with the BORDER_AGENT and UDP_FORWARD (or PLATFORM_UDP in case of RCP design) switches to build Border Router device. |
+| CHANNEL_MANAGER | OT_CHANNEL_MANAGER | 0 | Enables support for channel manager. | Enable this switch on devices that are supposed to request a Thread network channel change. This switch should be used only with an FTD build. |
+| CHANNEL_MONITOR | OT_CHANNEL_MONITOR | 0 | Enables support for channel monitor. | Enable this switch on devices that are supposed to determine the cleaner channels. |
+| COAP | OT_COAP | 0 | Enables support for the CoAP API. | Enable this switch if you want to control Constrained Application Protocol communication. |
+| COAPS | OT_COAPS | 0 | Enables support for the secure CoAP API. | Enable this switch if you want to control Constrained Application Protocol Secure (CoAP over DTLS) communication. |
+| COAP_OBSERVE | OT_COAP_OBSERVE | 0 | Enables support for CoAP Observe (RFC7641) API. |  |
+| COMMISSIONER | OT_COMMISSIONER | 0 | Enables support for Commissioner. | Enable this switch on device that is able to perform Commissioner role. CMake swtich: . |
+| COVERAGE | OT_COVERAGE | 0 | Enables the generation of code-coverage instances. |  |
+| CHILD_SUPERVISION | OT_CHILD_SUPERVISION | 0 | Enables support for child supervision. | Enable this switch on a parent or child node with custom OpenThread application that manages the supervision, checks timeout intervals, and verifies connectivity between parent and child. Read more about the child supervision: https://openthread.io/guides/build/features/child-supervision. |
```

Comment:
I would avoid having the link completely spelled out to ensure the column width isn't too big, instead link the text directly:

See [Child Supervision](https://openthread.io/guides/build/features/child-supervision) to learn more.